newdle/client/src/locales/es/messages.js

```diff
@@ -0,0 +1 @@
u would like to invite:":"Your newdle was created. You can now send the following to everyone you would like to invite:","Your participants will receive an e-mail asking them to register to their preference. Once the newdle is created, you will be shown a link you can share with anyone else you wish to invite.":"Your participants will receive an e-mail asking them to register to their preference. Once the newdle is created, you will be shown a link you can share with anyone else you wish to invite.","Your session expired":"Your session expired","by {author}":["by ",["author"]],"finished":"finished","newdle is Open Source Software":"newdle es Software de c\xF3digo abierto","newdle logo":"newdle logo","ongoing":"ongoing","originally created in the <0>{newdleTz}</0> timezone":["originally created in the <0>",["newdleTz"],"</0> timezone"],"switch back to the <0>{defaultUserTz}</0> timezone":["switch back to the <0>",["defaultUserTz"],"</0> timezone"],"{0, plural, one {# participant will be e-mailed:} other {# participants will be e-mailed:}}":[["0","plural",{one:["#"," participant will be e-mailed:"],other:["#"," participants will be e-mailed:"]}]],"{0, plural, one {+# more} other {+# more}}":[["0","plural",{one:["+","#"," more"],other:["+","#"," more"]}]],"{0, plural, one {You replied with # timeslot} other {You replied with # timeslots}}":[["0","plural",{one:["You replied with ","#"," timeslot"],other:["You replied with ","#"," timeslots"]}]],"{0, plural, one {{1} out of # participant answered} other {{2} out of # participants answered}}":[["0","plural",{one:[["1"]," out of ","#"," participant answered"],other:[["2"]," out of ","#"," participants answered"]}]],"{0, plural, zero {No participants registered} one {# participant registered} other {# participants registered}}":[["0","plural",{zero:"No participants registered",one:["#"," participant registered"],other:["#"," participants registered"]}]],"{0} will take place on:":[["0"]," will take place on:"],"{availableCount, plural, one {# available participant} other {# available participants}}":[["availableCount","plural",{one:["#"," available participant"],other:["#"," available participants"]}]],"{hours, plural, one {{hours} hour} other {{hours} hours}}":[["hours","plural",{one:[["hours"]," hour"],other:[["hours"]," hours"]}]],"{hours}h {minutes}min":[["hours"],"h ",["minutes"],"min"],"{numSlots, plural, one {<0>#</0> slot selected } other {<1>#</1> slots selected } =0 {You haven't selected any slots yet}}":[["numSlots","plural",{one:["<0>","#","</0> slot selected "],other:["<1>","#","</1> slots selected "],0:"You haven't selected any slots yet"}]],"{numberOfTimeslots, plural, one {{numberOfAvailable} out of # option chosen} other {{numberOfAvailable} out of # options chosen}}":[["numberOfTimeslots","plural",{one:[["numberOfAvailable"]," out of ","#"," option chosen"],other:[["numberOfAvailable"]," out of ","#"," options chosen"]}]],"{title} will take place on:":[["title"]," will take place on:"]};
```

Comment:
Delete this file and run `make` to get the latest js dependencies - the latest linguijs update switched from .js to .jsm so I updated the gitignore...